routing/conditions.rst

```diff
@@ -12,34 +12,37 @@ define arbitrary matching logic, use the ``conditions`` routing option:
 
     .. code-block:: yaml
 
+        # config/routes.yaml
         contact:
-            path:     /contact
-            defaults: { _controller: AcmeDemoBundle:Main:contact }
-            condition: "context.getMethod() in ['GET', 'HEAD'] and request.headers.get('User-Agent') matches '/firefox/i'"
+            path:       /contact
+            controller: 'App\Controller\DefaultController::contact'
+            condition:  "context.getMethod() in ['GET', 'HEAD'] and request.headers.get('User-Agent') matches '/firefox/i'"
 
     .. code-block:: xml
 
+        <!-- config/routes.xml -->
         <?xml version="1.0" encoding="UTF-8" ?>
         <routes xmlns="http://symfony.com/schema/routing"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:schemaLocation="http://symfony.com/schema/routing
                 http://symfony.com/schema/routing/routing-1.0.xsd">
 
             <route id="contact" path="/contact">
-                <default key="_controller">AcmeDemoBundle:Main:contact</default>
+                <default key="_controller">App\Controller\DefaultController::contact</default>
                 <condition>context.getMethod() in ['GET', 'HEAD'] and request.headers.get('User-Agent') matches '/firefox/i'</condition>
             </route>
         </routes>
 
     .. code-block:: php
 
+        // config/routes.php
         use Symfony\Component\Routing\RouteCollection;
         use Symfony\Component\Routing\Route;
 
         $collection = new RouteCollection();
         $collection->add('contact', new Route(
             '/contact', array(
-                '_controller' => 'AcmeDemoBundle:Main:contact',
+                '_controller' => 'App\Controller\DefaultController::contact',
```

Comment:
It's minor, and not worth delaying things because it's such a not important format, but in PHP, using the `[DefaultController::class, 'contact']` is friendlier.